app/main.py

```diff
@@ -2,24 +2,66 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(
+            self,
+            min_amount: int,
+            max_amount: int
+    ) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, instance: "Visitor", name: str) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self, instance: "Visitor", owner: object) -> object:
+        return getattr(instance, self.protected_name)
+
+    def __set__(self, instance: "Visitor", value: int) -> None:
+        if not isinstance(value, int):
+            raise TypeError()
+        if not (self.min_amount <= value <= self.max_amount):
+            raise ValueError()
+        setattr(instance, self.protected_name, value)
 
 
 class Visitor:
-   pass
+
+    def __init__(self, name: str, age: int, weight: int, height: int) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class SlideLimitationValidator(ABC):
-    pass
+    def __init__(self, age: int, weight: int, height: int) -> None:
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class ChildrenSlideLimitationValidator(SlideLimitationValidator):
-    pass
+    age = IntegerRange(4, 14)
+    weight = IntegerRange(20, 50)
+    height = IntegerRange(80, 120)
 
 
 class AdultSlideLimitationValidator(SlideLimitationValidator):
-    pass
+    age = IntegerRange(14, 60)
+    weight = IntegerRange(50, 120)
+    height = IntegerRange(120, 220)
 
 
 class Slide:
-    pass
+    def __init__(self, name: str,
+                 limitation_class: SlideLimitationValidator) -> None:
+        self.name = name
+        self.limitation_class = limitation_class
+
+    def can_access(self, visitor: Visitor) -> bool:
+        try:
+            self.limitation_class(visitor.age, visitor.weight, visitor.height)
+        except ValueError or TypeError:
```

Comment:
When you catch multiple exceptions, they should be put into a tuple, not separated by `or`. Correct this line to `except (ValueError, TypeError):`.